charts/logentries/Chart.yaml

```
@@ -2,10 +2,8 @@ name: logentries
 apiVersion: v1
 description: Helm chart for deploying an instance of logentries as a DaemonSet on all nodes of a K8s cluster.
 icon: https://d1cnss1t6ao97n.cloudfront.net/mstatic/3eb3f9c/content/uploads/2015/10/logentries-logo.png
-version: 1.2.3
+version: 1.2.4
```

```suggestion
version: 1.3.0
```
...its a feature release

---

Nothing changed for this chart, only removed maintainer :) Hesham marked his name everywhere 😄